ssl/s3_both.cc

```diff
@@ -666,7 +666,7 @@ class CipherScorer {
  public:
   CipherScorer(uint16_t group_id)
       : aes_is_fine_(EVP_has_aes_hardware()),
-        security_128_is_fine_(group_id != SSL_CURVE_CECPQ2) {}
+        security_128_is_fine_(!is_post_quantum_group(group_id)) {}
```

Comment:
From what I can tell, it's used to evaluate cipher suites to find the minimum acceptable security for both client and server. It's relevant here, because, presumably, if you are using a PQ key exchange, you are concerned with a quantum adversary and so using AES-128 is not acceptable.